<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forget_Dialog</class>
 <widget class="QDialog" name="forget_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="ID_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>49</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IDnumber_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> 身份证号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入本人身份证号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comfirm_pushButton">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>105</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordRe_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>144</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backGround.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>ID_lineEdit</zorder>
  <zorder>IDnumber_label</zorder>
  <zorder>label</zorder>
  <zorder>comfirm_pushButton</zorder>
  <zorder>passwordRe_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
